example/mysql/views/layouts/main.tt

```diff
@@ -17,13 +17,13 @@
 
 <!-- Nav Content -->
 <div class="collapse navbar-toggleable-xs" id="nav-content">
-<a class="navbar-brand" href="#">Manage company users</a>
+<a class="navbar-brand" href="#"><% settings.appname %></a>
 <ul class="nav navbar-nav">
 <li class="nav-item">
-<a class="nav-link" href="/people">Home</a>
+<a class="nav-link" href="<% request.uri_base %>">Home</a>
 </li>
 <li class="nav-item">
-<a class="nav-link" href="/people/add">Add</a>
+<a class="nav-link" href="/add">Add</a>
```

Comment:
Seems like using /people/add is the overall simplest thing here. Ideally this would work totally dynamically for other table types, but that's not needed for just this sample code.

I think the "get /add" route just makes things more difficult overall for the future when the user wants to add more simplecrud interfaces. .